src/fixtures/base.js

```diff
@@ -0,0 +1,16 @@
+// @ts-check
+import { test as base } from '@playwright/test';
+import { Application } from '../pages/Application';
+
+export const test = base.extend({
```

Comment:
can you please setup JSDoc so that intellisense (codecomplition in tests) works for JS code?